server/routes/products.js

```diff
@@ -21,6 +21,10 @@ products
   // return the id's of products related to the product specified
 products
   .route('/:product_id/related')
-  .get()
+  .get(((req, res) => {
+    getRelated((err, data) => {
```

Comment:
no error first callbacks! .catch block will do all error handling. 

this should be: 
getRelated()
  .then(data => {
something
})
.catch(err => {
something
})